### PR TITLE
Integrate Freshchat SDK

### PR DESCRIPTION
**Pull request by MuthuFD**
## Description

1. Added `react-native-freshchat-sdk` to the sample project
2. Intialised Freshchat SDK on app launch
3. Added button to show chat screen

### CODE REVIEW

File: Sample/App.js [hunk 1]
```diff
@@ -45,6 +47,9 @@ const App: () => Node = () => {
             backgroundColor: isDarkMode ? Colors.black : Colors.white,
           }}>
         </View>
+        <View style={{marginTop: 4 , padding: 10}} >
```
Comment (rajasegar-c) [inline Sample/App.js]: Is there a better way to add CSS other than inline styles here?

File: Sample/freshchat_utils.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    Freshchat,
+    FreshchatConfig,
+    FaqOptions,
+    ConversationOptions,
+    FreshchatUser,
+    FreshchatMessage,
+    FreshchatNotificationConfig
+} from 'react-native-freshchat-sdk';
+
+appId = 'c69641e9-8a85-4da1-858e-77169b0c76a7';
```
Comment (rajasegar-c) [inline Sample/freshchat_utils.js]: Should we include sensitive info here in the code, or should we pass it via environment variables?

**Review by rajasegar-c — CHANGES_REQUESTED** [review]

File: Sample/App.js (hunk 1 above)
Comment (MuthuFD) [reply Sample/App.js]: Style was not needed. We have remove it.

File: Sample/freshchat_utils.js (hunk 2 above)
Comment (MuthuFD) [reply Sample/freshchat_utils.js]: Removed all sensitive informations. Thanks.

**Pull request merged**